apps/files_sharing/lib/sharedstorage.php

```
@@ -55,6 +55,10 @@ public function __construct($arguments) {
 		$this->ownerView = $arguments['ownerView'];
 	}
 
+	private function init() {
+		Filesystem::initMountPoints($this->share['uid_owner']);
```

At this point we can no longer handle that gracefully, so I let it bubble up